### PR TITLE
Add shims for deprecated FormHelper methods

### PR DESCRIPTION
**Pull request by bancer**

### CODE REVIEW

File: Lib/Shim.php [hunk 1]
```diff
@@ -133,6 +133,14 @@ class Shim {
 	 */
 	const VALIDATE_PROPERTY = 'Shim.warnAboutValidateProperty';
 
+	/**
+	 * The name of the shim that checks the usage of deprecated since 3.4.0
+	 * FormHelper::input() and FormHelper::inputs().
+	 *
+	 * @var string
```
Comment (dereuromark) [inline Lib/Shim.php]: That we dont need I think.

Comment (bancer) [reply Lib/Shim.php]: Okay, that was added automatically by IDE.

**Comment by dereuromark** [comment]
I actually wanted to remove 2.x branch in favor of cake2 one.
I will merge into cake2, tag a new release and then remove the old invalid branch.

**Comment by bancer** [comment]
Okay. I think it was not fully ready to be merged though. I will add more tests.

**Pull request merged**